javascript-typing-tutor/main.js

```diff
@@ -1,28 +1,78 @@
-var $letterNodes = document.querySelectorAll('.phrase h1 span');
-var $completionText = document.querySelector('.completion-text');
+var $phraseH1 = document.querySelector('.phrase h1');
+var $letterNodes = $phraseH1.children;
+
+var phrase;
+var phraseList = [
+  "i don't know how to handle capital letters lol",
```

Comment:
You can check if the current character is uppercased or lowercased, and if it was uppercase, you would check the `shiftKey` property on the event object. If its value is `false`, they typed the lowercase, not the upper case 🤓 

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/shiftKey